docker-compose.override.yml

```diff
@@ -48,6 +50,7 @@ services:
       SOLR_CORES: $SOLR_CORES
     volumes:
       - ./files/init_solr.sh:/docker-entrypoint-initdb.d/10-init.sh
+
```

Comment:
haha, this is here cos I accidentally put the `files` volume in the wrong place :p